### ISSUE TITLE
Calendar Issues 9999

### ISSUE THREAD

**Issue by jackywongcw** [patchthecode/JTAppleCalendar#765]
IMPORTANT - BEFORE CREATING YOUR ISSUE PLEASE READ THE FOLLOWING:

1. PLEASE STATE THE VERSION NUMBER you are using.'
7.1.5
2. Are you using the latest version of JTAppleCalendar? Latest version is currently 7.1.5
Yes
3. PROVIDE ENOUGH INPORMATION SO I CAN RECREATE THE PROBLEM.
I've sent an email to you with this title as the subject.

Using ios 10. Please do look into it. I'm posting here to get your attention 🙏

**Comment by patchthecode** [comment]
What issues are you experiencing?

**Comment by jackywongcw** [comment]
A very random and rare chance of Calendar not being refreshed on viewWillAppear.

I have 2 buttons, one that change the calendar to 6 rows (Month view), one that change to 1 row (week view)
On viewWillAppear, most of the time it shows correct calendar with 6 rows, but sometimes just one row, and the only way to fix this is to change to week view, then back to month view.

While I'm on month view with the correct calendar showing, i can swipe to next month, upon clicking the monthview, it'll show me the current month.

But this correct behavior is not working when the incorrect calendar is occurred. clicking on monthview, wont refresh the calendar.

On month/week view clicked:
// change number of rows
        numberOfRows = 6

//reload calendar
        CalendarView.reloadData()

// scroll calendar
        CalendarView.scrollToDate(todaysDate, triggerScrollToDateDelegate: true, animateScroll: false)

**Comment by patchthecode** [comment]
is your project already secret? is it possible you can send me an empty app with the problem you are experiencing?

**Comment by jackywongcw** [comment]
Yes, i'm unable to show anything here. However I've sent you screenshots of the error via email, could you take a look? Unable create an empty app for now, even if i do, it may not have the issue, as the issue is appearing randomly on a low chance. 

Would showing you the code helps? Again, I can only do it via email, sorry for the inconvenience :/

**Comment by patchthecode** [comment]
@jackywongcw is it possible you can join me here? https://gitter.im/patchthecode/JTAppleCalendar
chat here is too slow
